From fd33bfeab9c2af84b2fd8fdb8a10d9f4f02be1f9 Mon Sep 17 00:00:00 2001
From: Steve Loughran <stevel@apache.org>
Date: Wed, 7 Sep 2016 12:19:05 +0100
Subject: [PATCH 2068/2848] HADOOP-13541 explicitly declare the Joda time
 version S3A depends on. Contributed by Stevel
 Loughran

Conflicts:
	hadoop-project/pom.xml

(cherry picked from commit 7fdfcd8a6c9e2dd9b0fb6d4196bc371f6f9a676c)

Change-Id: Icd91ac34b866492f7ffdb1f93c2483f01661380c
---
 hadoop-project/pom.xml          |    8 +++++++-
 hadoop-tools/hadoop-aws/pom.xml |    4 ++++
 2 files changed, 11 insertions(+), 1 deletion(-)

diff --git a/hadoop-project/pom.xml b/hadoop-project/pom.xml
index 8cb7df8..0bcb6ea 100644
--- a/hadoop-project/pom.xml
+++ b/hadoop-project/pom.xml
@@ -79,6 +79,7 @@
     <findbugs.version>3.0.0</findbugs.version>
 
     <tomcat.version>${cdh.tomcat.version}</tomcat.version>
+    <joda-time.version>2.9.4</joda-time.version>
 
     <!-- define the Java language version used by the compiler -->
     <javac.version>${javaVersion}</javac.version>
@@ -970,7 +971,12 @@
        <artifactId>bcprov-jdk16</artifactId>
        <version>1.46</version>
        <scope>test</scope>
-      </dependency>
+     </dependency>
+     <dependency>
+        <groupId>joda-time</groupId>
+        <artifactId>joda-time</artifactId>
+        <version>${joda-time.version}</version>
+     </dependency>
       <dependency>
         <groupId>com.microsoft.windowsazure.storage</groupId>
         <artifactId>microsoft-windowsazure-storage-sdk</artifactId>
diff --git a/hadoop-tools/hadoop-aws/pom.xml b/hadoop-tools/hadoop-aws/pom.xml
index e110e99..6be24e3 100644
--- a/hadoop-tools/hadoop-aws/pom.xml
+++ b/hadoop-tools/hadoop-aws/pom.xml
@@ -263,6 +263,10 @@
       -->
     </dependency>
     <dependency>
+      <groupId>joda-time</groupId>
+      <artifactId>joda-time</artifactId>
+    </dependency>
+    <dependency>
       <groupId>com.amazonaws</groupId>
       <artifactId>aws-java-sdk-sts</artifactId>
       <scope>compile</scope>
-- 
1.7.9.5

